specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-10-01/examples/TenantsGet.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+     "api-version": "2019-01-01",
+     "billingProfileId": "dd21e577-ec81-45ec-b08a-622213737978",
+	 "billingAccountId": "0d49b9e9-dfe7-536e-ede4-bae1d63adbb7"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/billingAccounts/0d49b9e9-dfe7-536e-ede4-bae1d63adbb7/billingProfiles/dd21e577-ec81-45ec-b08a-622213737978/providers/Microsoft.Consumption/tenants",
```

Comment:
Your id currently references the collection itself, we want to mimic a collection api who has unique resources in it. Your id should end with the 'name' of the resource, in your case 'tenant1'
"id": "...Microsoft.Consumption/tenants/tenant1",